server/commands/documentPermanentDeleter.js

```diff
@@ -7,6 +7,14 @@ import parseAttachmentIds from "../utils/parseAttachmentIds";
 const log = debug("commands");
 
 export async function documentPermanentDeleter(documents: Document[]) {
+  const index = documents.findIndex((doc) => !doc.deletedAt);
+
+  if (index !== -1) {
+    throw new Error(
+      `Cannot permanently delete ${documents[index].id} document. Please delete it and try again.`
+    );
+  }
```

Comment:
```suggestion
  const activeDocument = documents.find((doc) => !doc.deletedAt);

  if (activeDocument) {
    throw new Error(
      `Cannot permanently delete ${activeDocument.id} document. Please delete it and try again.`
    );
  }
```

This should work and be a little clearer to follow